storybookjs/storybook
# When timestamps are enabled, it actually checks them before applying changes

Issue: Fixing PR I did #1039.

## What I did
I accidentally added curly brace in wrong line. Because of that timestamps and debounce were actually not working properly, and first time user started typing text event still happened, even though it wasn't supposed to happen.


## How to test
See #1039. Using the same examples (set timestamps true, and debounce to { wait: 1000, leading: true}.
Before this update first initial update would still kick in. After this update, debounce will work properly.
